### ISSUE TITLE
Launching multiple instances of installer on Windows

### ISSUE THREAD

**Issue by olliefr** [getinsomnia/insomnia#3160]
**Describe the bug**
It is possible to start multiple instances of installer on Windows. I don't think it should be that way.
After I had downloaded the installer for Windows, I accidentally clicked it a few times, 

**To Reproduce**
1. Download the installer for Windows.
2. Double click on downloaded file at least six times in rapid succession.
3. Observe multiple instances of installer starting up.
4. Observe them proceeding in parallel. _Some_ might fail with an error.

**Expected behavior**
After the installer process had started and until it exits, further attempts to start another installer should either be ignored, or a message displayed, informing the user that the installer is already running.

**Screenshots**
Version info:
![image](https://user-images.githubusercontent.com/967105/110250945-d0c74e80-7f86-11eb-90f9-3343fce2eb32.png)

**Desktop (please complete the following information):**
 - OS: Windows
 - Installation Method: installer `.exe`, downloaded from Insomnia website
 - App Version v2021.1.0

**Additional context**
This is not a major issue, just a nice touch. Insomnia looks and feels well polished, so I thought you care about the detail, hence the bug report. Cheers! ))

**Comment by nijikokun** [comment]
Thanks for raising, going to close as this is out of scope for us as it's an upstream issue with [`electron-builder`](https://github.com/electron-userland/electron-builder) which we use for building the installer. If you feel that this issue was closed in error, feel free to open with more information!

**Comment by dimitropoulos** [comment]
@nijikokun do you have particular electron-builder issue handy just to close the loop on this?
